design/prototype-version-7/task-list/README.md

```diff
@@ -0,0 +1,140 @@
+- When users clicked on the link to “Add and edit your questions“ tasks, and added their questions, the grey highlighted tag on the far right of the row changed to a light blue tag that reads “in progress“ as shown below. Users were unsure how to move the tag to “complete“
+
+<br>
+
+![Step “1. Create your form” tasks. Screenshot](../screenshots/003-1-Task-list-page-tags-focus-create-your-form.png)
+*Step “1. Create your form” followed by 4 rows of tasks below, from version 6 of the prototype that we tested*
+
+<br>  
+<br>  
+
+## What we changed and why
+
+- Added a purple highlighted tag labelled “DRAFT“ for when the form is in draft and a green highlighted tag labelled “LIVE“ for when the form has been made live. 
+- Changed the link text from “Review summary page and add declaration“ to “Add a declaration for people to agree to“. This made the task and action for the form creator clear and specific. This content improvement made it clearer that the declaration is for people who are completing the form to agree to. 
+
+<br>
+
+![New step “1. Create your form” and list of tasks. Screenshot](../screenshots/003-create-form-create-form-statuses-focus.png)
+*Updated version for Step “1. Create your form” with 4 rows of tasks below and tags on the far right of each.*
+
+<br>  
+
+- Changed title for the second step from “2. Set form responses“ to "2. Set email address for completed forms“
+
+<br>
+
+![New step “2. Set email address for completed forms” and tasks. Screenshot](../screenshots/003-create-form-set-email-statuses-focus.png)
+*Updated version for Step "2. Set email address for completed forms“ with 2 rows of tasks below and tags on the far right of each.*
+
+<br>  
+
+- Changed step title from “3. Get your form ready to go live“ to “3. Provide privacy and contact details“
+- Edited the link text for the first task from “Provide a link to your privacy information“ to “Provide a link to privacy information for this form”
+- Changed the link text on second row from “Provide link to your accessibility statement“ to “Provide contact details for support“. This task was introduced so that form creators can provide at least one way for people to get help if they get stuck while filling in their form.
+
+<br>
+  
+![New step “3. Provide privacy and contact details” and tasks. Screenshot](../screenshots/003-create-form-privacy-contact-statuses-focus.png)
+*Updated version for Step “3. Provide privacy and contact details“ with 2 rows below and tags on the far right of each.*
+
+<br>  
+
+- Changed step title from “4. Publish your form“ to “4. Make your form live“.
+- Removed the final task ”How to publish the form on GOV.UK“ because the guidance to publish a form on GOV.UK was provided in the [“Make your form live“](design/prototype-version-7/screenshots/701-make-your-form-live.png) page.
```

Comment:
Is this supposed to link to a screenshot elsewhere on the page, or should it open the image file? 
This would open the image file: 
```suggestion
- Removed the final task ”How to publish the form on GOV.UK“ because the guidance to publish a form on GOV.UK was provided in the [“Make your form live“](../screenshots/701-make-your-form-live.png) page.
```

Reply:
It should open the make your form live screenshot, which can be found on the prototype version 7 read me